error_reporting/unit_tests/__init__.py

```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
```

Comment:
No thanks on these empty lines (my editor won't even save a file with empty lines it that end, it vacuums them up during save)